lib/models/ExtractionField.js

```diff
@@ -0,0 +1,8 @@
+class ExtractionField {
+    constructor(bodyFieldName, activityFieldName) {
```

Comment:
_bodyFieldName_ is the path to get the field and _activityFieldName_ is where to get it (body, headers...)?

Reply:
I remember now you explained to me yesterday, but I think it is confusing if it is not too hard to change. what about:
origRequestFieldName and resultActivityFieldName, or anything else that emphasises that the first is the place to take from and the second is the place to put in the activity

Reply:
Excellent comment. Originally when I first wrote it, the fields were `from` and `to` but what you suggested is exactly right.